wikimedia/wikipedia-ios
# Resign first responder before previewing to fix iOS14 push animation

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T258211

### Notes
* I'm not sure why this was happening to begin with, nor why this seems to fix it 🤷‍♀️  I tested on iOS 14 iPhone and iPad devices and this seems fixed. I also tested on iOS13 and iOS12 iPhone and iPad devices and didn't notice any regressions.

### Test Steps
1. Go through an editing flow. The animation hitch is on iOS 14, iPhones, when pushing the Edit Preview screen as well as when popping from Edit Preview. Confirm pushing and popping is as expected now and doesn't do the weird double-push / double-pop animation.
2. Feel free to edit an article on the Test language all the way through to publishing to be sure nothing gets wonky with saving an edit (though I already tested this).

### Screenshots/Videos *(if applicable)*
See Phab ticket for gif of the problem.
